up42/asset.py

```diff
@@ -93,24 +96,30 @@ def stac_items(self) -> pystac.ItemCollection:
 
     def update_metadata(
         self,
-        title: Optional[str] = None,
-        tags: Optional[List[str]] = None,
+        title: Union[Optional[str], object] = NOT_PROVIDED,
+        tags: Union[Optional[List[str]], object] = NOT_PROVIDED,
         **kwargs,
     ) -> dict:
         """
         Update the metadata of the asset.
 
         Args:
-            title: The title string to be assigned to the asset.
-            tags: A list of tag strings to be assigned to the asset.
+            title: The title string to be assigned to the asset. No value will keep the existing title.
+            tags: A list of tag strings to be assigned to the asset. No value will keep the existing tags.
 
         Returns:
             The updated asset metadata information
         """
+        if kwargs:
```

Comment:
@dustgalactic @kodamaaiko if the usage of kwargs arguments was never documented and doesn't work anyway for normal users, can we just drop them?

Reply:
Makes sense to me.

Reply:
I dont know the full implication, but if it's not documented and users arent using them, I say yes, lets drop.